exporter/splunkhecexporter/metricdata_to_splunk.go

```diff
@@ -228,9 +228,9 @@ func createEvent(timestamp pdata.Timestamp, host string, source string, sourceTy
 
 }
 
-func populateLabels(fields map[string]interface{}, labelsMap pdata.StringMap) {
-	labelsMap.Range(func(k string, v string) bool {
-		fields[k] = v
+func populateAttributes(fields map[string]interface{}, attributeMap pdata.AttributeMap) {
+	attributeMap.Range(func(k string, v pdata.AttributeValue) bool {
+		fields[k] = v.StringVal()
```

Comment:
Added link in https://github.com/open-telemetry/opentelemetry-collector/issues/3815